routers/api/v1/org/team.go

```diff
@@ -63,3 +62,112 @@ func GetTeamRepos(ctx *context.APIContext) {
 	}
 	ctx.JSON(200, repos)
 }
+
+// CreateTeam api for create a team
+func CreateTeam(ctx *context.APIContext, form api.CreateTeamOption) {
+	if !ctx.Org.Organization.IsOrgMember(ctx.User.ID) {
+		ctx.Error(403, "", "Must be an organization member")
+	}
+	team := &models.Team{
+		OrgID:       ctx.Org.Organization.ID,
+		Name:        form.Name,
+		Description: form.Description,
+		Authorize:   models.ParseAccessMode(form.Permission),
+	}
+	if err := models.NewTeam(team); err != nil {
+		if models.IsErrTeamAlreadyExist(err) {
+			ctx.Error(422, "", err)
+		} else {
+			ctx.Error(500, "NewTeam", err)
+		}
+		return
+	}
+
+	ctx.JSON(201, convert.ToTeam(team))
+}
+
+// EditTeam api for edit a team
+func EditTeam(ctx *context.APIContext, form api.EditTeamOption) {
+	if !ctx.User.IsUserOrgOwner(ctx.Org.Team.OrgID) {
+		ctx.Error(403, "", "Must be an organization owner")
+		return
+	}
+	team := &models.Team{
+		ID:          ctx.Org.Team.ID,
+		OrgID:       ctx.Org.Team.OrgID,
+		Name:        form.Name,
+		Description: form.Description,
+		Authorize:   models.ParseAccessMode(form.Permission),
+	}
+	if err := models.UpdateTeam(team, true); err != nil {
+		ctx.Error(500, "EditTeam", err)
+		return
+	}
+	ctx.JSON(200, convert.ToTeam(team))
+}
+
+// DeleteTeam api for delete a team
+func DeleteTeam(ctx *context.APIContext) {
+	if !ctx.User.IsUserOrgOwner(ctx.Org.Team.OrgID) {
+		ctx.Error(403, "", "Must be an organization owner")
+		return
+	}
+	if err := models.DeleteTeam(ctx.Org.Team); err != nil {
+		ctx.Error(500, "DeleteTeam", err)
+		return
+	}
+	ctx.Status(204)
+}
+
+// GetTeamMembers api for get a team's members
+func GetTeamMembers(ctx *context.APIContext) {
+	if !models.IsOrganizationMember(ctx.Org.Team.OrgID, ctx.User.ID) {
```

Comment:
The corresponding Github endpoint (`GET /teams/:id`) is only accessible to organization members (even if there are public members of the team), so I believe that is what we want to do here.